DicomRtImportExport/Logic/vtkSlicerDicomRtReader.cxx

```diff
@@ -816,8 +817,8 @@ void vtkSlicerDicomRtReader::vtkInternal::LoadRTPlan(DcmDataset* dataset)
   }
   else
   {
-    vtkErrorWithObjectMacro( this->External, "LoadRTPlan: Fraction Scheme is absent");
-    return;
+    vtkWarningWithObjectMacro( this->External, "LoadRTPlan: Fraction Scheme is absent");

```

Comment:
Is this an error that fraction scheme is absent?
If not then it is better to log it with a debug macro.

Reply:
Fixed